geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/EnvironmentCommand.scala

```diff
@@ -24,16 +25,65 @@ class EnvironmentCommand(parent: JCommander) extends Command(parent) with LazyLo
 
   // TODO accumulo environment?
   override def execute(): Unit = {
-    if (params.sfts == null && params.converters == null) {
-      // default - list all
-      listSfts()
+    if (params.sfts == null && params.converters == null && !params.listSfts && !params.listConverters && !params.describeSfts && !params.describeConverters) {
+      // default - print out parameters
       println
+      parent.usage(command)
+    } else if (params.listSfts){
```

Comment:
seems like there's a lot of duplication here. couldn't it be rewritten as a single block? e.g. 
```
else {
  if (params.listConverters) foo
  if (params.listSfts) bar
  // etc
}
```